static_src/actions/service_actions.js

```diff
@@ -146,10 +147,13 @@ const serviceActions = {
     return Promise.resolve(serviceInstance);
   },
 
-  errorCreateInstance(err) {
+  errorCreateInstance(error) {
+    const { response } = error;
+    const safeError = (response && response.data) || { code: 500 };
```

Comment:
@jcscottiii For your second comment, `error_code` needn't be present, the store checks for the existence of one and returns the generic 500 error message if it isn't present.